src/api/index.js

```diff
@@ -1,15 +1,7 @@
 const express = require('express');
 
-const emojis = require('./emojis');
-
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  res.json({
-    message: 'API - 👋🌎🌍🌏',
-  });
-});
-
-router.use('/emojis', emojis);
+// routers will be imported here
```

Comment:
Please add //TODO : add routed in here  in 

This way we can search for todos in the code and implement them (writing a comment like this will most likely be forgotten)